src/Types/ByteArray.js

```diff
@@ -0,0 +1,64 @@
+// From instance for Array in prelude
```

Comment:
This should prefix all functions that are unchanged reuse of Prelude functions. For others we do need tests.